src/composables/queries/useUserStudentDataQuery.js

```diff
@@ -3,18 +3,18 @@ import { storeToRefs } from 'pinia';
 import { useAuthStore } from '@/store/auth';
 import { computeQueryOverrides } from '@/helpers/computeQueryOverrides';
 import { fetchDocById } from '@/helpers/query/utils';
-import { USER_STUDENT_DATA_QUERY_KEY } from '@/constants/queryKeys';
+import { USER_CHILD_DATA_QUERY_KEY } from '@/constants/queryKeys';
 import { FIRESTORE_COLLECTIONS } from '@/constants/firebase';
 
 /**
- * User student data query.
+ * User child data query.
  *
  * @TODO: Evaluate wether this query can be replaced by the existing useUserDataQuery composable.
  *
  * @param {QueryOptions|undefined} queryOptions – Optional TanStack query options.
  * @returns {UseQueryResult} The TanStack query result.
  */
-const useUserStudentDataQuery = (queryOptions = undefined) => {
+const useUserChildDataQuery = (queryOptions = undefined) => {
```

Comment:
How come there's two useUserChildDataQuery? You created a new file with the same name.